release-23.2: roachtest: skip pgjdbc and hibernate test

rafiss opened the pull request:
A dependency was pulled from the package repository. On the master branch we are updating the pgjdbc version under test so it doesn't use this package, but for older branches that creates a lot of extra work, so we'll just skip it instead.

fixes https://github.com/cockroachdb/cockroach/issues/127261
fixes https://github.com/cockroachdb/cockroach/issues/127341
fixes https://github.com/cockroachdb/cockroach/issues/127257
fixes https://github.com/cockroachdb/cockroach/issues/127237
Release justification: test only change
Release note: None

rafiss committed:
roachtest: skip pgjdbc test

A dependency was pulled from the package repository. On the master
branch we are updating the pgjdbc version under test so it doesn't use
this package, but for older branches that creates a lot of extra work,
so we'll just skip it instead.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/127446)
<!-- Reviewable:end -->

rafiss commented:
blathers backport 24.1 23.1 24.1.3-rc 23.2.9-rc 23.1.24-rc

fqazi reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/127446#-:-O25rSefAzTGU4O5HHxq:bnfp4nl)* status: <span>:shipit:</span> complete! 1 of 0 LGTMs obtained (waiting on @DarrylWong and @vidit-bhat)



<!-- Sent from Reviewable.io -->

rafiss committed:
roachtest: skip hibernate test

A dependency was pulled from the package repository. On the master
branch we are updating the pgjdbc version under test so it doesn't use
this package, but for older branches that creates a lot of extra work,
so we'll just skip it instead.

Release note: None

rafiss committed:
roachtest: skip activerecord

We will skip this until the upstream tests are stabilized.

Release note: None
